src/System.Net.Sockets/tests/FunctionalTests/SendFile.cs

```diff
@@ -154,6 +154,7 @@ public void SendFile_Synchronous(IPAddress listenAt, bool sendPreAndPostBuffers,
                 client.SendFile(filename, preBuffer, postBuffer, TransmitFileOptions.UseDefaultWorkerThread);
 
                 client.LingerState = new LingerOption(true, LingerTime);
```

Comment:
>Does Shutdown simply mean "unlimited amount of linger time"?

Shutdown() will eventually call into unmanaged Interop.Winsock.shutdown(handle, (int)how), and take as many time as it needs to shutdown the connection. 

While set the LingerTime is, setting the amount of time to remain connected after calling the Socket.Close method if data remains to be sent. 

I think here use Shutdown() is better, because the number of LingerTime is arbitrary. My Z240 is fast, and probably only need 2 seconds to send all data, while the Dell workstation need 10 seconds. Let's say I set LingerTime =10 here, and all tests can pass. What if someone is using super-slow computer and needs longer than 10 seconds? That cannot solve the issue.